frontend/integration-tests/views/crud.view.ts

```diff
@@ -58,7 +58,8 @@ export const actions = Object.freeze({
   edit: 'Edit',
   delete: 'Delete',
 });
-export const actionForLabel = (label: string) => $(`[data-test-action="${label}"]`);
+export const actionForLabel = (label: string) =>
+  $(`[data-test-action="${label}"]:not(.pf-m-disabled)`);
```

Comment:
I think this change broke the alertmanager test, which looks for a disabled menu item

Reply:
It shouldn't affect that test case. It uses a separate selector for the disabled kebab button. I'll dig a little bit more.

Reply:
Nevermind, I found the issue.